data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/validator/GetAttestation.java

```diff
@@ -55,25 +55,26 @@ public GetAttestation(final ChainDataProvider provider, final JsonProvider jsonP
   @OpenApi(
       path = ROUTE,
       method = HttpMethod.GET,
-      summary = "Get an unsigned attestation of a slot from the current state.",
+      summary = "Get an unsigned attestation for a slot from the current state.",
       tags = {TAG_VALIDATOR},
       queryParams = {
         @OpenApiParam(
             name = SLOT,
-            description = "The non-finalized slot to create the attestation for (uint64)",
+            description = "Slot to create the attestation for.",
```

Comment:
dangling preposition...

Reply:
if its finalized you'll get an error, which is why i specified that here...